recipes/libpqxx/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        bin_path = os.path.join("bin", "test_package")
```

Comment:
Surround with `if not tools.cross_building(self.settings):` to support cross building.

Reply:
I did not know it. Thank you.

Reply:
Fixed: f19309d743b6c3be46e5a50cbe1dbd3a4e1f50f4